src/general/replicalimits/src_test.rego

```diff
@@ -51,23 +51,39 @@ empty = {
 }
 
 review(replicas) = output {
+  replicas > 0
```

Comment:
nit: It might be nice to support a test where `spec.replicas` is specifically set to `0`?

Reply:
The rule just below with `replicas == 0` is what tests that, by blanking out `spec` (that's what the whole PR is about).  I agree that having an additional test that explicitly checks `spec.replicas=0` would be good, but it would require a rather large refactoring of the source-to-tests logic to allow for `review(replicas)` to return two different objects when `replicas=0`.  In fact, that very idea (a Rego function that returns multiple values for the same input) is strictly forbidden by Rego.  So the whole thing would need to be reworked from scratch.

Reply:
That said, I see no evidence that `src_test.rego` is even being read by anything -- it certainly doesn't seem to be generating the tests under `library/general/replicalimits`, so I suppose I could add such a test manually and it would likely still pass all the CI tests.

Reply:
Hi @skaven81 - The Rego tests under `src/general/replicalimits/` do not build the KRM tests under `library/general/replicalimits/`, however they are run by the`CI / Unit test on {}` using the `opa test` command for each folder under `src/` - this is basically the same as running `./test.sh` locally.